addon/mixins/select.js

```diff
@@ -9,11 +9,11 @@ export default Mixin.create({
       this.set("label", label);
     } else if (typeof checkValue === "string" || typeof value === "number") {
       this.set("label", checkValue);
-    } else if (checkValue.text) {
-      this.set('label', checkValue.text);
-    } else if (checkValue.formattedTitle) {
+    } else if (checkValue.get('text')) {
```

Comment:
🍹  You're using double quotes everywhere else here, so change this to be consistent